ambry-clustermap/src/test/java/com.github.ambry.clustermap/MockReplicaId.java

```diff
@@ -104,4 +108,12 @@ public void cleanup() {
     }
     replicaDir.delete();
   }
+
+  /**
+   * Mark a replica as down or up.
+   * @param status if true, marks the replica as down; if false, marks it as up.
+   */
+  public void markReplicaDownStatus(boolean status) {
```

Comment:
while the documentation makes it clear, just looking at the variable name `status` and its definitions of `true` and `false`, makes it seem out of sync. Perhaps the variable is better named `isDown` (or `isUp` with values switched)?